src/components/MessagesList/MessagesGroup/Message/MessagePart/ReactionsWrapper.vue

```diff
@@ -43,16 +49,18 @@
 			</div>
 		</NcPopover>
 
-		<!-- all reactions button -->
-		<NcButton
-			v-if="showControls"
-			size="small"
-			:title="t('spreed', 'Show all reactions')"
-			:aria-label="t('spreed', 'Show all reactions')"
-			@click="showAllReactions = true">
-			<IconHeartOutline :size="15" />
-		</NcButton>
-		<span v-else class="reaction-button--thumbnail" />
+		<template v-if="!isSplitViewEnabled">
+			<!-- all reactions button -->
+			<NcButton
+				v-if="showControls"
+				size="small"
+				:title="t('spreed', 'Show all reactions')"
+				:aria-label="t('spreed', 'Show all reactions')"
+				@click="showAllReactions = true">
+				<IconHeartOutline :size="15" />
+			</NcButton>
+			<span v-else class="reaction-button--thumbnail" />
```

Comment:
Didn't we reserve a place for this?

no hover | hover
-- | --
<img width="313" height="147" alt="Image" src="https://github.com/user-attachments/assets/bfb700f1-1a04-42b9-926b-55c938cdf9cc" /> | <img width="315" height="147" alt="Image" src="https://github.com/user-attachments/assets/3c960982-1807-48b8-b042-ca5bec0235aa" />